src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -46,7 +48,10 @@ public LinuxAppServiceFileLogger(string logFileName, string logFileDirectory, IF
         public int MaxFileSizeMb { get; set; } = 10;
 
         // Maximum time between successive flushes (seconds)
-        public int FlushFrequencySeconds { get; set; } = 30;
+         public int MaxFlushFrequencySeconds { get; set; } = 30;
+
+        // Flush Frequency Growth Factor
+        public int FlushFrequencyGrowthFactor { get; set; } = 2;
```

Comment:
I think you can just remove this and hardcode the factor to 2 inline where it's used.

Reply:
Coded inline thanks